Projects/Project1/app/src/main/java/com/example/mcresswell/project01/fragments/FitnessDetailsFragment.java

```diff
@@ -118,7 +122,13 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
             mSensorManager = (SensorManager) getActivity().getSystemService(getActivity().SENSOR_SERVICE);
             mStepCounter = mSensorManager.getDefaultSensor(Sensor.TYPE_STEP_COUNTER);
-            m_tvstepCount.setText(String.format(Locale.US, INT_FORMAT + STEPS, m_numberOfSteps));
```

Comment:
are you sure this line of code compiles? You are trying to use String.format to format a String (m_numberOfSteps) using an integer formatter.